src/client/js/cart-main.js

```diff
@@ -0,0 +1,23 @@
+/**
+ *
+ *  Online store PWA sample.
+ *  Copyright 2017 Google Inc. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+import init from './cart.js';
+import {Router} from './router';
```

Comment:
Router init call is now in app.js

Reply:
So should the code here be like this?

```
import initApp from './app.js';
import init from './cart.js';

initApp();
init();

Reply:
Yes please

Reply:
Actually now it got moves to page-init. Please check other -main files for an example, e.g. category-main.js.